master/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -46,6 +46,11 @@ kubectl expose --namespace=advanced-policy-demo deployment nginx --port=80
 
 #### Check using calicoctl
 
+  > **Note:**
+  >
+  > This requires the [calicoctl tool to be configured]({{site.baseurl}}/{{page.version}}/reference/calicoctl/setup/etcdv2).
+  > For example: `export ETCD_ENDPOINTS=http://10.96.232.136:6666`
```

Comment:
I think the Vagrant demo default IP for etcd is `172.24.197.101`? Can you check and use whatever that value is?